### DISCUSSION TITLE
Could not build wheels for mysqlclient, which is required to install pyproject.toml-based projects

### DISCUSSION THREAD

**Discussion by akshayjain3450** [apache/superset#27049] [Q&A / Help]
In the latest master: when I try `make install` then, I face this issue:  ERROR: Could not build wheels for mysqlclient, which is required to install pyproject.toml-based projects. What is the issue here?

**Comment by akshayjain3450** [comment]
Resoved at my end.

**Reply by sergioruizsan** [reply]
@akshayjain3450 can you explain how did you fix it? Facing the same issue
